### ISSUE TITLE
toggle button does not expand itself

### ISSUE THREAD

**Issue by gregjozk** [nvaccess/nvda#7640]
### Steps to reproduce:

1. open firefox
2. go to any page which has expandable button (e.ge nvda mainpage on groups.io)
3. restart NVDA
4. try to expand the first button so you can choose a view (messages or topics)

### Expected behavior:

button should expand itself.
If I want this behaveour, I must restart firefox.

### Actual behavior:

NVDA does not interact with this type of button, so this button is still collapsed after pressing space or enter even though NVDA says expanded.

### System configuration:
NVDA version: latest next

NVDA Installed or portable: installed

Windows version:
w10 1703
Name and version of other software in use when reproducing the issue:
Firefox 54 55 and 56

### Other questions:

Does the issue still occur after restarting your PC?
after make steps described above.
Have you tried any other versions of NVDA?

I think that this refers to #7269 and #7535.

**Comment by leonardder** [comment]
What makes you think that this is NVDA specific?

**Comment by Adriani90** [comment]
I cannot reproduce this under NVDA 2017.4, Firefox 57 and Windows 10 Build 16299.64.

**Comment by Adriani90** [comment]
I have tried it on github as it has been described above.

**Comment by feerrenrut** [comment]
@gregjozk Can you still reproduce this issue?

**Comment by gregjozk** [comment]
No. It seems that one of other fixes fixed also this problem.

regards, Jožef

2017-12-20 4:41 GMT+01.00, Reef Turner <notifications@github.com>:

**Comment by gregjozk** [comment]
Hello,

I can reproduce it after nvda was updated from next 14728,d60e4746 to
next 14735,47900869
It seems that a problem exists after updating NVDA or am I wrong?
I also observed that now I must refrash (nvda+f5) most webpages if I
want to use them.

regards and thx for your good work.

2017-12-20 17:07 GMT+01.00, J.G <jozko.gregorc@guest.arnes.si>:

**Comment by feerrenrut** [comment]
I am not able to reproduce this with Firefox Version `57.0.2` and NVDA Version: `next-14753,cae9ca42`

@gregjozk Could you restart your computer and try again?

**Comment by gregjozk** [comment]
Hi,

here are 2 logs. one of them contains log, where I reproduced this problem on FF (esr 52.5.2), other log is from nvda, where all works correctly. however since I recently also have reproduced it very rarely, this ticket can be closed. If I find reliable way to reproduce it, I will comment and report.

I tested it on today's next snapshot.

thx for your hard work.

[nvda errors.zip](https://github.com/nvaccess/nvda/files/1580255/nvda.errors.zip)

**Comment by feerrenrut** [comment]
I had a look at the logs, and nothing stood out to me. As suggested, will close this issue for now.
